spec/registrant_spec.rb

```diff
@@ -0,0 +1,39 @@
+require './lib/registrant'
+
+RSpec.describe Registrant do
+  before(:each) do
+    @registrant_1 = Registrant.new('Bruce', 18, true )
+    @registrant_2 = Registrant.new('Penny', 15 )
+  end
+
+  describe "Initialize" do
+    it "exists" do
+      expect(@registrant_1).to be_an_instance_of(Registrant)
+      expect(@registrant_2).to be_an_instance_of(Registrant)
```

Comment:
Redundant testing here. For things like instantiating and checking attributes, we really only have to test it once and then we can assume it will work for all Registrant instances.